pkgs/top-level/all-packages.nix

```diff
@@ -1600,6 +1600,8 @@ with pkgs;
 
   ocs-url = libsForQt5.callPackage ../tools/misc/ocs-url { };
 
+  openbugs = callPackage ../applications/science/machine-learning/openbugs { };
```

Comment:
```suggestion
  openbugs = pkgsi686Linux.callPackage ../applications/science/machine-learning/openbugs { };
```

Reply:
I think this will make the package only available for 32-bit arch, bit it is mainly used in 64-bit beefy computers. I do not think this is the correct way as the package is targeted for 64-bit use also. Please correct me if I am wrong.

Reply:
pkgsi686Linux means "build 32-bit binaries, even on 64-bit", which is exactly what we want here, right?

Reply:
Yes that's correct. let me fix it. Thank you.